vignettes/development_process.Rmd

```diff
@@ -0,0 +1,39 @@
+---
+title: "Development Process"
+output: 
+  rmarkdown::html_vignette
+    
+vignette: >
+  %\VignetteIndexEntry{Development Process}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Step-by-Step Guide
+
```

Comment:
This seems to be the only heading in the document. Thus, I'd remove it and just have the title displayed.